mlflow/pytorch/__init__.py

```diff
@@ -1099,3 +1125,79 @@ def print_auto_logged_info(r):
     autolog.__doc__ = autolog.__doc__.replace("MIN_REQ_VERSION", str(MIN_REQ_VERSION)).replace(
         "MAX_REQ_VERSION", str(MAX_REQ_VERSION)
     )
+
+
+def load_checkpoint(model_class, run_id=None, epoch=None, global_step=None):
```

Comment:
Is this a public API?

Reply:
Yes. I will add it to __all__

Reply:
added.

Reply:
How do I use this API? Any example?

Reply:
This is a demo notebook https://e2-dogfood.staging.cloud.databricks.com/?o=6051921418418893#notebook/2173893049403456

After model checkpoint is logged, you can call `load_checkpoint` API to get the checkpoint model

Reply:
can you add an example in the docstring?

Reply:
can you add a `Returns` section in the docstring?

Reply:
Returns section and example code are added